example/lib/pages/navigation.dart

```diff
@@ -1419,6 +1462,45 @@ class _NavigationPageState extends ExamplePageState<NavigationPage> {
                       }),
                 ]),
           )),
+      Card(
+        child: ExpansionTile(
+            enabled: _isAutoScreenAvailable,
+            title: const Text('Auto view'),
+            collapsedTextColor: getExpansionTileTextColor(
+                !_navigatorInitialized || _navigationViewController == null),
+            collapsedIconColor: getExpansionTileTextColor(
+                !_navigatorInitialized || _navigationViewController == null),
+            children: <Widget>[
+              ElevatedButton(
+                onPressed: () => _setMapTypeForAutoToSatellite(),
+                child: const Text('Set map type to satellite'),
+              ),
+              ElevatedButton(
+                onPressed: () => _moveCameraForAuto(),
+                child: const Text('Move camera'),
+              ),
+              ElevatedButton(
+                onPressed: () => _addMarkerForAuto(),
+                child: const Text('Add marker'),
+              ),
+              ExampleSwitch(
```

Comment:
Why enablng turn-by-turn events is under Auto View? Should it be under Navigation view?

Reply:
Done.